docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.3'
+services:
+  jekyll:
+    build: .
+    ports:
+      - '4000:4000'
+    volumes:
+      - $PWD/docs:/site/docs
+      - $PWD/.git:/site/.git
```

Comment:
This looks great! The only other problem I can see is that this assumes the user is rendering their docs to a `docs` folder, but a user can modify that behavior with `outputDir` in the `docs-ts.json`. Should this be included in the `README`?

Reply:
Good point!

Just checked, the following would work:


`docker-compose.yml`
```
version: '3.3'
services:
  jekyll:
    build: .
    ports:
      - '${PORT:-4000}:4000'
    volumes:
      - $PWD/${OUTPUT_DIR:-docs}:/site/docs
      - $PWD/.git:/site/.git
```

It introduces the optional environment variables `OUTPUT_DIR` and `PORT` with their defaults `docs` and `4000`.

So I can run `BUILD_DIR=my-docs PORT=4001 docker-compose --file ....`.
And internally they'll be mapped to `docs` and `4000` again. So no need for any changes in the `Dockerfile`.

If you agree I can change that and add a note to the README.

Reply:
LGTM! Excellent work on this @thought2!